### ISSUE TITLE
About general prompts

### ISSUE THREAD

**Issue by Lincoln20030413** [Ephemeral182/ECCV24_T3-DiffWeather#3]
Thanks for your fromer replies! I'm sorry but I'm a little confused about this part in your paper:

![7c460d7fc1cd9e043c47f61f9f2b3be](https://github.com/user-attachments/assets/b186c64f-0055-4967-ae2c-cf046e42ef6d)
What's the query in the cross-attention when generating general prompts. The words "the general prompts form the queries" really confuse me a lot. Aren't we going to generate prompt？

**Comment by Ephemeral182** [comment]
I apologize for the confusion. But what I mean here is that for the normal prompt, there is no way to constrain it to control the background. So we use the normal prompt as the query of the cross attention to constrain it in combination with the features of depth anything. The normal prompt here is first generated by learnable parameters.

**Comment by Lincoln20030413** [comment]
Thanks. Does it means that you initialize the general prompts and generate the updated general prompts using the initialized general prompts as query? I first think that this part needn't being trained and now maybe it also need being trained.

**Comment by Ephemeral182** [comment]
Yes, your understanding is correct, this part requires training cross attention.

**Comment by Lincoln20030413** [comment]
Thanks for your patient replies!
